### PR TITLE
Added Navigation and grid

### PR DESCRIPTION
**Pull request by rahul31agrawal**
Hi guys,
I have added Navigation & Grid to my component library, Please give your reviews.
Thanks

### CODE REVIEW

File: cssFiles/navigation.css [hunk 1]
```diff
@@ -0,0 +1,70 @@
+.navigation-C{
+    min-height: 100px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    gap: .5rem;
+    margin: auto;
+    box-shadow: rgba(0, 0, 0, 0.02) 0px 1px 3px 0px, rgba(27, 31, 35, 0.15) 0px 0px 0px 1px;
```
Comment (harshitpaliwal95) [inline cssFiles/navigation.css:8]: can create variable for box-shadow too

**Comment by harshitpaliwal95** [comment]
looks good ☑️

File: cssFiles/navigation.css (hunk 1 above)
Comment (rahul31agrawal) [reply cssFiles/navigation.css:8]: Ok, I will do the correction

File: cssFiles/grid.css [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    max-width: 36rem;
+    display: grid;
+    grid-gap: 1rem;
+    margin: auto;
+  }
+  
+  .grid-items {
+    padding: 1rem;
+    text-align: center;
+    border: 1px solid black;
```
Comment (shubhmehra) [inline cssFiles/grid.css:33]: can you use this color fro  your root  instead of directly defining them?

**Review by shubhmehra — APPROVED** [review]

File: cssFiles/grid.css (hunk 2 above)
Comment (rahul31agrawal) [reply cssFiles/grid.css:33]: sure, 
thanks

**Pull request merged**